lib/command.rb

```diff
@@ -12,6 +14,24 @@ def self.synopsis
 
       def execute
         with_target_vms(nil, {:single_target=>true}) do |machine|
+
+	  options = {}
+          opts = OptionParser.new do |o|
+	    o.banner = "Usage: vagrant adbinfo [options]"
+            o.separator ""
+
+            o.on('box-version', 'box-version', 'Prints the version of Virtual Machine') do |boxversion|
+	      options[:boxversion] = boxversion;
+	    end
+          end
+          
+          # parse the options
+          argv = parse_options(opts)
+	  if argv and argv.include? 'box-version' then
```

Comment:
Unnecessary `then` keyword. 
Also, prefer `&&` over `and`(check <a href='http://stackoverflow.com/a/10648796/1120530' target='_blank'>nice reason here</a>).